lms/templates/learner_dashboard/program_details.html

```diff
@@ -13,14 +13,13 @@
 
 <%block name="js_extra">
 <%static:require_module module_name="js/learner_dashboard/program_details_factory" class_name="ProgramDetailsFactory">
-ProgramDetailsFactory({
-    programData: ${program_data | n, dump_js_escaped_json}
```

Comment:
If you check-in this change, the frontend will no longer get any data from backend anymore. Maybe we should revert?